whitepaper/Makefile

```diff
@@ -37,3 +37,7 @@ latex:
 clean:
 	@rm -rf $(BUILDDIR)
 	@echo "Removed $(BUILDDIR) directory"
+
+github:
+	ghp-import -n $(BUILDDIR)
```

Comment:
shouldn't this be preceded by @ ?

Reply:
also, isn't this copying full directory?

only relevant artifact should be main.pdf i think?

Reply:
> shouldn't this be preceded by @ ?

Fixed


> also, isn't this copying full directory?
> 
> only relevant artifact should be main.pdf i think?

It's copying the full directory. The command only accepts passing directories, no single files. If we want just to deploy to GitHub pages just ``main.pdf``, the generated file should be in a separate folder Example: /_build/pdf/main.pdf

Reply:
probably good enough